index.html

```diff
@@ -147,6 +161,7 @@ <h1>I’m Abhishek <br>
             </div>
         </div>
     <div class="describe">
+        <div class="abt-self">
```

Comment:
- [ ] Your about me description is taking too little space so i think it will be better if the about me description takes 40% and the technologies part takes the remaining 60% of the page's width. You can achieve this using flexbox

FIgma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/193346070-4350c0c3-5cc2-44f3-84ad-d6f6c84c6eaf.png)


Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/193346175-ffdba5a0-e51a-4005-9202-9bc1c3b0b42f.png)

Reply:
at my screen it is working fine